server/controllers/wazuh-utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './wazuh-utils';
+export * from './ui.logs.controller';
```

Comment:
use kebab case for the name of the files with dot-separated for the type 
  `ui-logs.controller` 
  please change all new files names